lib/chef/resource/systemd_unit.rb

```diff
@@ -67,6 +67,9 @@ class SystemdUnit < Chef::Resource
                description: "The name of the unit file if it differs from the resource block's name.",
                introduced: "13.7"
 
+      property :sensitive, [ TrueClass, FalseClass ],
```

Comment:
This is actually there by default in all resources so it doesn't need to be defined again.

Reply:
Thanks! @tas50, I'll update it.